bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -239,6 +241,9 @@ protected boolean getEditMode() {
         if (rulesProperties.size() > 0) {
             customProperties.put(CUSTOM_RULE_PROPERTY_WRAPPER, rulesProperties);
         }
+        if (getSignerProperties().size() > 0) {
```

Comment:
Please add unit test cases for this change

Reply:
Done